apps/landing/features/home/HeadingSection/Result/index.tsx

```diff
@@ -0,0 +1,113 @@
+import { type FC } from 'react';
+import { Image, StyleSheet } from 'react-native';
+import { Button, Text, View } from '@walless/gui';
+import { Twitter } from '@walless/icons';
+
+interface Props {
+	waitlistNumber: number;
+}
+
+const ResultModal: FC<Props> = ({ waitlistNumber }) => {
+	return (
+		<View style={styles.container}>
+			<View style={styles.header}>
+				<Image
+					source={{ uri: '/img/walless-icon.svg' }}
+					style={styles.logoImage}
+				/>
+				<Text style={[styles.text, styles.title]}>
+					Yayyy! You&apos;re on the list!
+				</Text>
+				<Text style={styles.text}>
+					We&apos;re happy to have you here. You&apos;ll receive a confirmation
+					email shortly. Share this to your fellow explorers :)
+				</Text>
+			</View>
+
+			<View style={styles.resultContainer}>
+				<View style={styles.resultImageContainer}>
+					<Image
+						source={{ uri: '/img/sign-up-icon.svg' }}
```

Comment:
same as above, we never know if we would use this image again or not, but all the visual content should come from 1 place which is `resources`